### PR TITLE
Use element if available for native Navigator

### PR DESCRIPTION
**Pull request by MarcoPolo**
@keybase/react-hackers @chrisnojima

### CODE REVIEW

File: shared/router/meta-navigator.render.native.js [hunk 1]
```diff
@@ -14,10 +13,12 @@ export default class MetaNavigatorRender extends Component {
         initialRouteStack={routeStack.toJS()}
         configureScene={route => route.sceneConfig || Navigator.SceneConfigs.FloatFromRight}
         renderScene={(route, navigator) => {
+          const element = route.element
           const Module = route.component
           return (
             <View style={{flex: 1, marginTop: route.hideNavBar ? 0 : this.props.navBarHeight}}>
-              <Module {...route.props} />
+              {element}
+              {!element && Module && <Module {...componentAtTop.props}/>}
```
Comment (chrisnojima) [inline shared/router/meta-navigator.render.native.js]: not route.props?

Comment (MarcoPolo) [inline shared/router/meta-navigator.render.native.js]: whoops, just copied pasted that!

Comment (MarcoPolo) [inline shared/router/meta-navigator.render.native.js]: fixed

**Comment by chrisnojima** [comment]
:+1:

**Pull request merged**